bins/recipe-runner/src/runner.c

```diff
@@ -619,7 +626,7 @@ GglError runner(const RecipeRunnerArgs *args) {
     }
     default:
         GGL_LOGE("Failed to get proxy url from config.");
-        return ret;
+        break;
```

Comment:
If breaking doesn't cause program errors then this shouldn't be logged as an error. I'd prefer logging this as a warning and having a corresponding info statement to mention that the GG-lite instance will not proxy requests.

Reply:
This should probably exit. If we fail to talk with config, the only error that we should continue with is for missing value (since thats expected).

Reply:
Missing value is not currently reported as such. We need to find a way to communicate the exact error that the server faced to the client.

As of now, we only get `EVENTSTREAM_APPLICATION_ERROR` and there is no way to know what was the actual error. That is why the default code for now is like this.

Reply:
This MUST be kept as is for now till my next PR which will update the client code which will then be able to parse the error codes sent by the server and make a proper decision.